extension-impl/remoting-bolt/src/main/java/com/alipay/sofa/rpc/transport/bolt/BoltClientTransport.java

```diff
@@ -435,15 +416,19 @@ public void handleRpcRequest(SofaRequest request) {
 
     @Override
     public InetSocketAddress remoteAddress() {
+        Connection connection = connectionManager.getConnection(RPC_CLIENT, transportConfig, url);
```

Comment:
All like this line can use `fetchConnection()` instead.

Reply:
> All like this line can use `fetchConnection()` instead.

fixed